sdk/js/src/svm.ts

```diff
@@ -82,6 +88,175 @@ export async function constructSubmitBidInstruction(
   return ixSubmitBid;
 }
 
+export function getAssociatedTokenAddress(
+  owner: PublicKey,
+  tokenMintAddress: PublicKey,
+  tokenProgram: PublicKey
+): PublicKey {
+  return getAssociatedTokenAddressSync(
+    tokenMintAddress,
+    owner,
+    true,
+    tokenProgram,
+    ASSOCIATED_TOKEN_PROGRAM_ID
+  );
+}
+
+export function createAssociatedTokenAccountIdempotentInstruction(
+  owner: PublicKey,
+  mint: PublicKey,
+  payer: PublicKey = owner,
+  tokenProgram: PublicKey
+): [PublicKey, TransactionInstruction] {
+  const ataAddress = getAssociatedTokenAddress(owner, mint, tokenProgram);
+  const createUserTokenAccountIx = createAssociatedTokenAccountInstruction(
```

Comment:
why not just use this [helper](https://solana-labs.github.io/solana-program-library/token/js/functions/createAssociatedTokenAccountIdempotentInstruction.html)?